app/assets/scss/gov-au-ui-kit/_navigation.scss

```diff
@@ -195,9 +195,10 @@ Style guide: Navigation.6 Footer navigation
 .local-nav-toggle {
```

Comment:
We really shouldn't be editing the ui kit files. There is a corresponding `_navigation.scss` file in `scss`